src/features/components/templates/footer/index.tsx

```diff
@@ -46,4 +49,22 @@ export const MainFooter = () => {
     )
 }
 
+const getSocialButtons = (lang: string, is_eu: boolean, is_career: boolean, config: TAppConfig) => {
+    const overrideWithLang = (arr) =>
+        arr.map((button) =>
+            lang in specialLanguageUrls
+                ? button['aria-label'] in specialLanguageUrls[lang]
+                    ? { ...button, href: specialLanguageUrls[lang][button['aria-label']] }
+                    : button
+                : button,
+        )
+
+    let buttons = is_career ? socialButtonsCareers : is_eu ? socialButtonsEU : socialButtonsROW
+
+    buttons = buttons.filter((button) => !!config[button?.['aria-label']])
+    buttons = overrideWithLang(buttons)
+
+    return buttons
+}
```

Comment:
Can you explain this? What is happening here?

Reply:
same I'm so confused, I think we can make this simpler what is causing it to add this much complexity?

Reply:
we need to check region (eu or row), special langs has special links (need to change only 3 social links, not 6) for both row and eu regions, need to filter by growthBook config (is it enabled or not), also not forget to check and change links for career page. thats why you think that is so complicated.

Reply:
@hasan-deriv @mitra-deriv good question. This could have been explained in the PR description. Effectively, there is no complexity here, but let me explain how the algorithm works:

1) If a user is on a career-page -- he gets some links, if on other pages -- standard links. 

2) The links that are going to be returned first get checked if they are enabled in the config (upcoming feature)

3) Then links are overridden with some special-regional links (current feature request).